Jenkinsfile

```diff
@@ -108,14 +121,22 @@ node {
                     ../.venv/bin/pycodestyle legion
                     ../.venv/bin/pycodestyle tests
                     ../.venv/bin/pydocstyle legion
-    
+
                     export TERM="linux"
                     rm -f pylint.log
                     ../.venv/bin/pylint legion >> pylint.log || exit 0
                     ../.venv/bin/pylint tests >> pylint.log || exit 0
                     cd ..
+
+                    cd etl
+                    ../.venv/bin/pycodestyle etl
+                    ../.venv/bin/pycodestyle tests
+                    ../.venv/bin/pydocstyle etl
+                    ../.venv/bin/pylint etl >> pylint.log || exit 0
+                    ../.venv/bin/pylint tests >> pylint.log || exit 0
+                    cd ..
```

Comment:
*etl/pylint.log* needs to be archived and analyzed, in same way as *legion/pylint.log* is.